src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -189,77 +190,59 @@ public bool TryResolveFilePath(int resultId, int runIndex, string uriBaseId, str
                 return false;
             }
 
-            if (dataCache.FileDetails.ContainsKey(relativePath))
+            string solutionPath = this.GetSoltionPath();
+
+            // File contents embedded in SARIF.
+            bool hasHash = dataCache.FileDetails.ContainsKey(relativePath) && !string.IsNullOrEmpty(dataCache.FileDetails[relativePath].Sha256Hash);
```

Comment:
>ContainsKey [](start = 49, length = 11)

maybe u can change this for trygetvalue

Reply:
done

---
In reply to: [601278465](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r601278465) [](ancestors = 601278465)